.github/workflows/vast.yaml

```diff
@@ -859,6 +867,50 @@ jobs:
             docker tag tenzir/vast:latest "tenzir/vast:${release_tag}"
             docker push "tenzir/vast:${release_tag}"
           fi
+  docker-integrations:
+    needs:
+      - build-docker
+    if: github.event_name != 'workflow_dispatch'
+    name: Docker Integrations
+    runs-on: ubuntu-20.04
+    env:
+      DOCKER_BUILDKIT: 1
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Download Docker Image
+        uses: actions/download-artifact@v3
+        with:
+          name: vast_latest.tar
+      - name: Save Production Docker Image to Disk
+        run: |
+          docker load -i vast_latest.tar
+      - name: Start Docker Compose
+        run: |
+          docker-compose -f docker-compose.yml \
+                         up --no-build --detach
+      - name: Simple Test
+        run: |
+          docker-compose run vast status
+      - name: Ingest Test Data
+        run: |
+          docker cp "vast/integration/data/suricata/eve.json" tenzir-vast:/var/lib/vast
+          docker exec -t tenzir-vast vast import -r "/var/lib/vast/eve.json" suricata
```

Comment:
```suggestion
          docker-compose run --no-TTY vast import suricata < vast/integration/data/suricata/eve.json
```

Reply:
Latest commit fixes this. Thanks for pointing this.